gpytorch/lazy/non_lazy_tensor.py

```diff
@@ -80,6 +80,9 @@ def mul(self, other):
         else:
             return super(NonLazyTensor, self).mul(other)
 
+    def __getitem__(self, index):
+        return self.tensor.__getitem__(index)
+
```

Comment:
This also fails - because we may need to wrap it as a NonLazyTensor. I tried this earlier.

Reply:
Implemented the fix for this that we discussed offline